src/controllers/examples.ts

```diff
@@ -205,18 +208,13 @@ export const outOfBandRecordExample = {
   reusable: false,
 }
 
-export const CredentialExchangeRecordExample = {
-  _tags: {
+export const CredentialExchangeRecordExample: CredentialExchangeRecordProps = {
+  tags: {
     state: 'offer-sent',
     threadId: '82701488-b43c-4d7b-9244-4bb204a7ae26',
     connectionId: 'ac6d0fdd-0db8-4f52-8a3d-de7ff8ddc14b',
   },
-  metadata: {
-    '_internal/indyCredential': {
-      credentialDefinitionId: 'q7ATwTYbQDgiigVijUAej:3:CL:318187:latest',
-      schemaId: 'q7ATwTYbQDgiigVijUAej:2:Employee Badge:1.0',
-    },
-  },
+  role: CredentialRole.Issuer,
```

Comment:
`role` is now required https://github.com/openwallet-foundation/credo-ts/blob/e58ec5bd97043d57fcc3c5a4aee926943e6c5326/packages/core/src/modules/credentials/repository/CredentialExchangeRecord.ts#L19